### PR TITLE
added flag for normalisation

### PR DESCRIPTION
**Pull request by prottayCMT**

### CODE REVIEW

**Review by fmazzasc — APPROVED** [review]

**Review by mfaggin — APPROVED** [review]

**Comment by vkucera** [comment]
@prottayCMT @mfaggin @fmazzasc How are the two commits related?

File: PWGLF/TableProducer/Common/spvector.cxx [hunk 1]
```diff
@@ -389,12 +403,41 @@ struct spvector {
...
+          qyZDCA = qyZDCA / meansqyA;
+          qxZDCC = qxZDCC / meansqxC;
+          qyZDCC = qyZDCC / meansqyC;
+
+        } else {
+          qxZDCA = qxZDCA;
+          qyZDCA = qyZDCA;
+          qxZDCC = qxZDCC;
+          qyZDCC = qyZDCC;
+        }
```
Comment (vkucera) [inline PWGLF/TableProducer/Common/spvector.cxx:440]: This is useless.

**Comment by vkucera** [comment]
@prottayCMT Please avoid making pull requests containing completely unrelated code changes in the future. Also add a prefix with the name of the concerned PWG to indicate the scope of the PR.

**Comment by vkucera** [comment]
@mfaggin @fmazzasc Please try to avoid merging PRs containing completely unrelated code changes or with vague or misleading titles.
Merging this PR resulted in a single commit in the upstream master branch titled "added flag for normalisation" which gives no information about where the flag was added and completely misses any mention of the changes on the HF side.

**Pull request merged**